test-network-nano-bash/ca/ca_utils.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env sh
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+######################################################################################
+#  createEnrollment()
+#
+#  This is a convenience method for creating enrollments only
+#  Primary purpose it to create enrollment certificates for CAs
+######################################################################################
+
+function createEnrollment() {
+
+  local port=$1             # port of the CA used for creating the enrollment
+  local username=$2         # username of the registered user on the CA 
+  local password=$3         # password of the registered user on the CA
+  local orgname=$4          # name of the org (e.g. Org1, Org2)  can be blank
+  local component_dir=$5    # path of the component, this will be the directory where the artifacts will be created
+  local tlscert=$6          # tls cert for connecting to the CA
+
+  # Enroll the identity
+  fabric-ca-client enroll -d -u https://${username}:${password}@localhost:${port} --caname ca --mspdir "${component_dir}/msp" --tls.certfiles $tlscert
+
+  # Rename private key to mimic cryptogen
+  find ${component_dir} -type f -name '*_sk'  | sed -e 'p;s/\(.*\)\/\(.*\)$/\1\/priv_sk/' | xargs -n2 mv -v
+
+  # Rename the cacert to mimic cryptogen
+  mv ${component_dir}/msp/cacerts/localhost-${port}-ca.pem ${component_dir}/msp/cacerts/ca.${orgname:+$orgname.}example.com-cert.pem
+
+  # Set the cacert name and copy the config.json for NodeOU
```

Comment:
CA cert doesn't use NodeOUs right?

Reply:
You're right.  This is not needed for the admin cert.  Removing this line.